app/src/main/java/com/github/fribourgsdp/radio/SnapshotUtility.kt

```diff
@@ -0,0 +1,22 @@
+package com.github.fribourgsdp.radio
+
+import com.google.firebase.firestore.DocumentSnapshot
```

Comment:
Thank you for implementing this class, which makes the code reading a bit clearer when accessing the database snapshots.

Reply:
You're welcome